### PR TITLE
ISPN-15331 Upgrade Hibernate Search to 7.0.0.CR1

### PR DESCRIPTION
**Pull request by fax4ever**
https://issues.redhat.com/browse/ISPN-15331

### CODE REVIEW

File: client/hotrod-client/src/test/java/org/infinispan/client/hotrod/evolution/IndexSchemaNoDowntimeUpgradeTest.java [hunk 1]
```diff
@@ -363,147 +359,6 @@ void testAddNonAnalyzedIndexOnExistingField() {
...
-
-        remoteCacheManager.administration().reindexCache(CACHE_NAME);
-
-        // VERSION 2 - after reindex all entries are recognized (10)
-        // lowercase normalizer removed => the query is case-sensitive, also old entries are found
-        doQuery("FROM evolution.Model e WHERE e.normalizedField LIKE '%NORMALIZED%'", cache, 10);
-        // keyword analyzer => case-sensitive, whole text as one token, also old entries are found
-        doQuery("FROM evolution.Model e WHERE e.analyzedField : '*ANALYZED field*'", cache, 10);
-    }
-
```
Comment (yrodiere) [inline client/hotrod-client/src/test/java/org/infinispan/client/hotrod/evolution/IndexSchemaNoDowntimeUpgradeTest.java:506]: Was something not working?

**Review by yrodiere — APPROVED** [review]
LGTM, though I'm not sure why the tests were removed :shrug:

File: client/hotrod-client/src/test/java/org/infinispan/client/hotrod/evolution/IndexSchemaNoDowntimeUpgradeTest.java (hunk 1 above)
Comment (fax4ever) [reply client/hotrod-client/src/test/java/org/infinispan/client/hotrod/evolution/IndexSchemaNoDowntimeUpgradeTest.java:506]: https://infinispan.org/docs/stable/titles/query/query.html#update-index-schema_indexing-caches

**Comment by ryanemerson** [comment]
Thanks @fax4ever

**Comment by fax4ever** [comment]
Thank you Ryan!

**Pull request merged**